src/Lykke.Service.Qtum.Api.Core/Helpers/UnixTimeHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Lykke.Service.Qtum.Api.Core.Helpers
+{
+    public static class UnixTimeHelper
+    {
+        public static DateTime UnixTimeStampToDateTime(double unixTimeStamp)
+        {
+            // Unix timestamp is seconds past epoch
+            var dtDateTime = new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
It may be made static.

Reply:
do you mean move dtDateTime to static property?

Reply:
Yes, concrete DateTime 01.01.1970 may be static.